### PR TITLE
Polish the web client UI

### LINKED ISSUE #4: Polish the web client UI
# Objectives

Prettier UI and better code

### PR DESCRIPTION
**Pull request by air-sara**
closes #4

### CODE REVIEW

**Review by jmf1sh — CHANGES_REQUESTED** [review]
Looks good, two minor things:
* metrics/data should be in gitignore, those files should be removed and not be tracked
* I think that the activity-logger directory can be removed entirely (it's vestigal)

File: .gitignore [hunk 1]
```diff
@@ -27,6 +27,8 @@ node_modules/
 # Runtime data
 /data
 ./data
+/metrics
```
Comment (cloderic) [inline .gitignore:30]: This shouldn't be required anymore, the metrics directory doesn't exist anymore

File: web-client/package.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@cogment/cogment-js-sdk": "^1.26.0",
+    "@cogment/cogment-js-sdk": "^1.26.1",
```
Comment (cloderic) [inline web-client/package.json:6]: IIRC because of this update there's a tsignore somewhere that should be removable

**Review by cloderic — CHANGES_REQUESTED** [review]

**Review by cloderic — APPROVED** [review]
👍 looks good to me

**Review by jmf1sh — APPROVED** [review]

**Pull request merged**